HACKING.rst

```diff
@@ -223,3 +242,10 @@ variable annotations specified in `PEP-526`_ were introduced in Python
 
 .. _PEP-484: https://www.python.org/dev/peps/pep-0484/
 .. _PEP-526: https://www.python.org/dev/peps/pep-0526/
+
+.. [#fixture-list] This list of fixtures (with markup) can be
+   reproduced by running::
+
+     py.test-3 --fixtures -q | grep "^[^ ]" | grep -v no | sed 's/.*/* ``\0``/'``
+
+   in a xenial lxd container with python3-pytest installed.
```

Comment:
unrelated: when are we gonna start testing with a newer LTS by default?

Reply:
Travis [already tests against the latest PyPI packages by default](https://github.com/canonical/cloud-init/blob/46cf23c28812d3e3ba0c570defd9a05628af5556/.travis.yml#L16-L19). So in one sense, we already do.

That said, the reason for this constraint is not our upstream CI testing, it's because we backport the latest cloud-init upstream to xenial, and the unit tests run as part of the package build. This is why Travis has [a separate job](https://github.com/canonical/cloud-init/blob/46cf23c28812d3e3ba0c570defd9a05628af5556/.travis.yml#L46-L49) to test against the xenial versions of our dependencies. So in another sense, the _earliest_ LTS we want the tests (and cloud-init in general) to run on won't change until, at the earliest, xenial leaves standard support [in April 2021](https://wiki.ubuntu.com/XenialXerus/ReleaseNotes#Support_lifespan).